.github/checkgroup.yml

```diff
@@ -203,8 +202,6 @@ subprojects:
       #- "pl-cpu (ubuntu-20.04, lightning, 3.7, 1.9, oldest)"
       - "pl-cpu (windows-2022, lightning, 3.10, 1.12)"
       #- "pl-cpu (windows-2022, lightning, 3.7, 1.9, oldest)"
-      - "make-doctest (pytorch)"
-      - "make-html (pytorch)"
```

Comment:
Are there any implications here? What if the source code removes something that is referenced in the docs. This could generate a reference error in the docs build, and it would only show up on master.

Reply:
That's why I added https://github.com/Lightning-AI/lightning/pull/15581/files#diff-5ca41bd8852d2c309825fa76269fb299e7a77b075918a9c56fa1990c40e48f7bR122